### DISCUSSION TITLE
Updating tags on quay.io

### DISCUSSION THREAD

**Discussion by johnaohara** [Hyperfoil/Horreum#623] [General]
@jesperpedersen the Horreum Operator (https://github.com/Hyperfoil/horreum-operator/) default to image `quay.io/hyperfoil/horreum:latest`, however that image has not been updated for 8months now. 

During the release process we should also tag the release with `latest` so that the operator picks up the last released version by default

**Comment by jesperpedersen** [comment]
I have updated `latest` to point to 0.7.11
